caravel/bin/caravel

```diff
@@ -125,6 +125,35 @@ def refresh_druid():
             "[" + cluster.cluster_name + "]")
     session.commit()
 
+@manager.option(
+    '-p', '--prefix', default='data_',
```

Comment:
Well, first thing is that you should not install caravel on the same database of your data. Second the feature should be generic and not tied to your own usage :)